framework/package.json

```diff
@@ -45,7 +45,7 @@
 		"@liskhq/lisk-chain": "^0.4.0-alpha.0",
 		"@liskhq/lisk-codec": "^0.3.0-alpha.0",
 		"@liskhq/lisk-cryptography": "^4.0.0-alpha.0",
-		"@liskhq/lisk-db": "^0.3.0-debug.20-b44082d07ab158736d00f7fd1706ca72ca2c9d27",
+		"@liskhq/lisk-db": "^0.3.0-debug.20-5058ddc140c457740c275198deebe158b896f2d7",
```

Comment:
there are several places need the update. You can search with `"@liskhq/lisk-db"`

Reply:
Updated all places.